Plugins/Flow.Launcher.Plugin.BrowserBookmark/Commands/Bookmarks.cs

```diff
@@ -6,19 +6,19 @@ namespace Flow.Launcher.Plugin.BrowserBookmark.Commands
 {
     internal static class Bookmarks
     {
-        internal static bool MatchProgram(Bookmark bookmark, string queryString)
+        internal static MatchResult MatchProgram(Bookmark bookmark, string queryString)
         {
-            if (StringMatcher.FuzzySearch(queryString, bookmark.Name).IsSearchPrecisionScoreMet()) return true;
-            if (StringMatcher.FuzzySearch(queryString, bookmark.PinyinName).IsSearchPrecisionScoreMet()) return true;
```

Comment:
this search is not needed?

Reply:
I think the fuzzy search has contains the pinyin search feature, so this should be a duplicate one.

Reply:
oh ok. Makes sense. Should we remove the PinyinName property completely?

Reply:
Sounds good. I will remove it.